dts/bindings/serial/st,stm32-uart-base.yaml

```diff
@@ -58,3 +58,30 @@ properties:
       configured masked at boot (sm32wl55 for instance), preventing the device to wakeup
       the core from stop mode(s).
       Valid range: 0 - 31
+
+  de-enable:
+    type: boolean
+    description: |
+      Enabled activating an external transeiver through the DE signal.
```

Comment:
Might be good to add a word stating that configuring DE pin is user's responsibility.